docs/site/tutorials/authentication/Authentication-Tutorial.md

```diff
@@ -48,1298 +61,251 @@ permits access to the protected endpoint.
-
-1. In the `bearerAuth` value field, enter the token string you obtained earlier,
-   and press the `Authorize` button. This JWT token is now available for the
-   `/users/me` endpoint we will interact with next. Press the `Close` button to
-   dismiss the dialog.
-
-   ![](../../imgs/api_explorer_auth_token_dialog2.png)
-
-   {% include note.html content="The <b>Logout</b> button allows you to enter a new value; if needed." %}
-
-1. Scroll down to the `UserController` section to find `GET /users/me`
-
-   ![](../../imgs/api_explorer_usercontroller_section1.png)
-
-   Notice it has a **lock** icon and the other endpoints in this section do not.
-   This is because this endpoint specified an operation-level
-   `security requirement object` in the OpenAPI specification. (For details, see
-   the
-   [Specifying the Security Settings in the OpenAPI Specification](#specifying-the-security-settings-in-the-openapi-specification)
-   section.)
-
-1. Expand the `GET /users/me` section, and click on `Try it out`. There is no
-   data to specify, so simply click on `Execute`. The JWT token you specified
-   earlier was automatically placed in the `Authorization` header of the
-   request.
-
-   If authentication succeeds, the
-   [user profile](https://github.com/strongloop/loopback-next/blob/master/packages/security/src/types.ts)
-   of the currently authenticated user will be returned in the response. If
-   authentication fails due to a missing/invalid/expired token, an
-   [HTTP 401 UnAuthorized](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401)
-   is thrown.
-
-   The response contains a unique value in the `id` field (generated by the
-   database) and `name` field with the full user name:
-
-   ```json
-   {"id": "5dd6acee242760334f6aef65", "name": "User One"}
-   ```
-
-## Adding JWT Authentication to a LoopBack 4 Application
-
-In this section, we will demonstrate how `authentication` was added to the
-[loopback4-example-shopping](https://github.com/strongloop/loopback4-example-shopping)
-application using the
-[JSON Web Token (JWT)](https://en.wikipedia.org/wiki/JSON_Web_Token) approach.
+## Before we begin
 
-### Installing @loopback/authentication
-
-The `loopback4-example-shopping` application **already** has the
-`@loopback/authentication` dependency set up in its **package.json**
-
-It was installed as a project dependency by performing:
+Let’s download the Todo example and install the `@loopback/authentication-jwt`
+extension.
 
 ```sh
-npm install --save @loopback/authentication
+$ lb4 example todo
+$ cd loopback4-example-todo
+$ npm i --save @loopback/authentication @loopback/authentication-jwt
 ```
 
-### Adding the AuthenticationComponent to the Application
+## Step 1: Bind JWT Component in the Application
+
+In `src/application.ts`, bind the authentication components to your application
+class. Since we will be testing the JWT authentication using API Explorer, we
+will add the `addSecuritySpec()` function.
```

Comment:
> Since we will be testing the JWT authentication using API Explorer, we
will add the `addSecuritySpec()` function.

`addSecuritySpec()` is not needed anymore.